include/boost/url/bnf/parse.hpp

```diff
@@ -77,214 +96,182 @@ parse(
     error_code& ec,
     char ch) noexcept;
 
-/** @brief Parse a sequence of elements
+/** Parse beginning of a char range as a sequence of arbitrary BNF elements
 
    This function parses the string defined by the first chars in the
-   range of chars [@ref it, @ref end) into the Backus–Naur form (BNF)
-   elements [arg_0, arg_1, ..., arg_n] in @ref args.
+   range of chars [`it`, `end`) into the Backus–Naur form (BNF)
+   elements [`arg_0`, `arg_1`, ..., `arg_n`] in `args`.<br>
 
-   For each type Arg_i in [Arg_0, Arg_1, ..., Arg_n] in @ref Args, a
-   corresponding function overload
-   `bool parse(char const*& it, char const* const end, error_code&, Arg_i const& t)`
-   should be defined to consume the input from [@ref it, @ref end) into `arg_i`.
+   For each type `Arg_i` in [`Arg_0`, `Arg_1`, ..., `Arg_n`] in `Args`, a
+   corresponding function overload of `parse`
+   `bool parse(char const*& it, char const* const end, error_code&, Arg_i const& arg_i)`
+   should be defined to consume the input from [`it`, `end`) into `arg_i`.<br>
```

Comment:
This is way too technical and jargony. It needs to be written in a simpler fashion that ordinary users who don't read wg21 papers can grok.

Reply:
Right. I usually find it less ambiguous and easier to read something closer to formal mathematical symbolism but you're correct. Not many people will like that.